### PR TITLE
resolve duplicate attachedDebtInformation functions

### PR DESCRIPTION
**Pull request by simonnineteen90**
https://eaflood.atlassian.net/browse/SFI-1998

### CODE REVIEW

File: app/debt/save-debt-data.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
 const db = require('../data')
 
 const saveDebtData = (updatedDebtData, transaction) => {
-  const { paymentRequestId, attachedDate, debtDataId } = updatedDebtData
+  const { paymentRequestId, debtDataId } = updatedDebtData
```
Comment (AbidemiAdio) [inline app/debt/save-debt-data.js:4]: Any justification for not retrieving attachedDate from updatedDebtData as it was before?

Comment (simonnineteen90) [reply app/debt/save-debt-data.js:4]: Didnt realise I could retrive it and thought it needed redeclaring. I will amend.

Comment (simonnineteen90) [reply app/debt/save-debt-data.js]: resolved as discussed by renaming `updatedDebtData` to `debtData` for clarity

**Review by AbidemiAdio — APPROVED** [review]
QED

**Pull request merged**